spec/openapi.yaml

```diff
@@ -2534,7 +2534,7 @@ paths:
           -F "name=context1.png"
           "https://api.smartling.com/context-api/v2/projects/$smartlingProjectId/contexts"
 
-          
+
     get:
       summary: List contexts for the project
```

Comment:
List of visual context files within a Smartling project

Reply:
This line was changed automatically by editor (removed empty spaces on Save). I left it in PR intentionally because it requires to revert this change on each commit.
So I'm not going to fix here because it's VitaliyB's API.